src/hdx_cli/library_api/common/role_validator.py

```diff
@@ -0,0 +1,347 @@
+from typing import List, Optional, Union
+import re
+import uuid
+from pydantic import BaseModel
+
+from ..userdata.token import AuthInfo
+from ..common import rest_operations as rest_ops
+
+
+AVAILABLE_SCOPE_TYPE = ('user', 'pool', 'role', 'invite', 'org', 'project',
+                        'alterjob', 'batchjob', 'catalog', 'hdxstorage',
+                        'dictionary', 'dictionaryfile', 'function', 'table',
+                        'kafkasource', 'siemsource', 'kinesissource',
+                        'summarysource', 'transform', 'view')
+ALL_PERMISSIONS = ('user', 'pool', 'role', 'invite', 'org')
```

Comment:
Same comment as before. These things can be loaded dynamically.

Reply:
ALL_PERMISSIONS = ('user', 'pool', 'role', 'invite', 'org')

I used this information to obtain all permissions without repetition. 
By combining scope permissions (user + pool + role + invite + org), I can represent all possible permissions without duplication.

I'm considering creating a set with all permissions and then displaying the individual permissions without repetition.
make sense for you?